js-apps/meep-frontend/src/js/components/helper-components/nc-group.js

```diff
@@ -236,43 +256,52 @@ const NCGroup = ({ prefix, onUpdate, element }) => {
   // Retrieve field names
   var latencyFieldName = null;
   var latencyVarFieldName = null;
-  var throughputFieldName = null;
+  var latencyDistFieldName = null;
+  var throughputDlFieldName = null;
+  var throughputUlFieldName = null;
   var packetLossFieldName = null;
   switch (prefix) {
```

Comment:
Question - since we moved to a common structure for net.chars
What is the benefit of keeping this switch case of specific net.char types?

Reply:
Actually, this is a valid comment, the model used to be flattened, but it is not required anymore